src/common/transformations/src/transformations/common_optimizations/transpose_sinking.cpp

```diff
@@ -181,9 +181,10 @@ ov::pass::TransposeReduction::TransposeReduction() {
         if (!transpose_order || !reduction_axes)
             return false;
 
-        const auto& non_negative_axes = ov::util::normalize_axes(reduction->get_friendly_name(),
-                                                                 reduction_axes->cast_vector<int64_t>(),
-                                                                 reduction->get_input_partial_shape(0).rank());
+        const auto non_negative_axes =
```

Comment:
Consider to use const ref.

Reply:
Is not required as axes are created here.